### ISSUE TITLE
Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms)

### ISSUE THREAD

**Issue by secsilm** [tensorflow/tensorflow#7842]
I got the following error when I run `python cifar10_train.py`.

```
E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_dnn.cc:390] Loaded runtime CuDNN library: 5005 (compatibility version 5000) but source was compiled with 5105 (compatibility version 5100).  If using a binary install, upgrade your CuDNN library to match.  If building from sources, make sure the library loaded at runtime matches a compatible version specified during compile configuration.
F c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\kernels\conv_ops.cc:605] Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms)
```

Operating System: Windows 10
CUDA: `Cuda compilation tools, release 8.0, V8.0.44`
cuDNN: 5.1
tensorflow: 1.0.0

The output from `python -c "import tensorflow; print(tensorflow.__version__)"`

```
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:135] successfully opened CUDA library cublas64_80.dll locally
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:135] successfully opened CUDA library cudnn64_5.dll locally
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:135] successfully opened CUDA library cufft64_80.dll locally
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:135] successfully opened CUDA library nvcuda.dll locally
I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:135] successfully opened CUDA library curand64_80.dll locally
1.0.0
```

I have upgrade cudnn from 5.0 to 5.1. But it didn't work.

**Comment by mrry** [comment]
How did you install cuDNN 5.1? Is it possible that cuDNN 5.0 is still in your `%PATH%`?

**Comment by secsilm** [comment]
@mrry Yes, that is the problem. I reset the cuDNN PATH. Problem solved! Thank you!

**Comment by mrry** [comment]
It was a lucky guess :). Glad to hear it!

**Comment by XiaoYigwr** [comment]
When I use the keras==1.2.0, I have got the same problem. Fortunately,  the issue was solved after I upgrade the tensorflow from 1.2.0 to 1.3.0 .Maybe you can have a try
